alethzero/MainWin.cpp

```diff
@@ -1872,7 +1880,31 @@ void Main::on_send_clicked()
 			debugFinished();
 			Secret s = i.secret();
 			if (isCreation())
+			{
+				// If execution is a contract creation, add Natspec to
+				// a local Natspec LEVELDB
 				ethereum()->transact(s, value(), m_data, ui->gas->value(), gasPrice());
+				string src = ui->data->toPlainText().toStdString();
```

Comment:
To be honest, this whole file should be Split and properly abstracted and refactored but well from the way I understand it, any time spent on AZ is gonna be a waste.